design/experimental-new-features-in-modular.md

```diff
@@ -0,0 +1,178 @@
+
+### Proposal with single client
+
+In the case of single-client,
+
+<!-- markdownlint-disable MD033 -->
+<table>
+  <tr>
+    <th>Stable</th>
+    <th>Experimental</th>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+@azure/foo
+@azure/foo/api
+@azure/foo/rest
+</pre>
+</td>
+<td>
+<pre lang="typescript">
+@azure/foo/beta
+@azure/foo/beta/api
+@azure/foo/beta/rest
+</pre>
+</td>
+  </tr>
+</table>
+<!-- markdownlint-enable MD033 -->
+
+### Proposal with multi-client
+
+<!-- markdownlint-disable MD033 -->
+<table>
+  <tr>
+    <th>Stable</th>
+    <th>Experimental</th>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+@azure/foo
+@azure/foo/api
+@azure/foo/rest
+</pre>
+<pre lang="typescript">
+@azure/foo/clientA
+@azure/foo/clientA/api
+</pre>
+<pre lang="typescript">
+@azure/foo/clientB
+@azure/foo/clientB/api
+</pre>
+</td>
+<td>
+<pre lang="typescript">
+@azure/foo/beta
+@azure/foo/beta/api
+@azure/foo/beta/rest
+</pre>
+<pre lang="typescript">
+@azure/foo/beta/clientA
+@azure/foo/beta/clientA/api
+</pre>
+<pre lang="typescript">
+@azure/foo/beta/clientB
+@azure/foo/beta/clientB/api
+</pre>
+</td>
+  </tr>
+</table>
+<!-- markdownlint-enable MD033 -->
+
+### Proposal with multi-endpoint
+
+<!-- markdownlint-disable MD033 -->
+<table>
+  <tr>
+    <th>Stable</th>
+    <th>Experimental</th>
+  </tr>
+  <tr>
+    <td>
+      <pre lang="typescript">
+@azure/foo
+@azure/foo/api
+</pre>
+<pre lang="typescript">
+@azure/foo/clientA
+@azure/foo/clientA/api
+@azure/foo/rest/clientA
+</pre>
+<pre lang="typescript">
+@azure/foo/clientB
+@azure/foo/clientB/api
+@azure/foo/rest/clientB
+</pre>
+</td>
+<td>
+<pre lang="typescript">
+@azure/foo/beta
+@azure/foo/beta/api
+</pre>
+<pre lang="typescript">
+@azure/foo/beta/clientA
+@azure/foo/beta/clientA/api
+@azure/foo/beta/rest/clientA
+</pre>
+<pre lang="typescript">
+@azure/foo/beta/clientB
+@azure/foo/beta/clientB/api
+@azure/foo/beta/rest/clientA
+</pre>
+</td>
+  </tr>
+</table>
+<!-- markdownlint-enable MD033 -->
+
+## SDK version policies  
+
+1. Bump minor stable version with `/beta` subpath export if preview any service side/client side breaking/non-breaking features.
+1. Release a beta package that reflects the code changes before GA the preview features.
+1. Release a major beta version if we are previewing any service-side/client-side breaking features to prepare for the GA.
+1. If we are planning to GA only service side features that include breaking changes from both service side and client side and client side features have not ready for GA yet, we will release a major version beta package with `/beta` subpath export where the preview service version is in the stable code and client-side feature is in the `/beta` subpath export.
```

Comment:
Could you provide some samples especially for 4 & 5 ?

Reply:
Okay, let me think how to provide the samples.